index.bs

```diff
@@ -1143,6 +1146,25 @@ The following constraints define when a [=/transaction=] can be
 
 </div>
 
+To <dfn export>cleanup transactions</dfn> for an [=event loop=], run
```

Comment:
Could you call this "cleanup Indexed Database transactions"? Or do you want HTML to call it "cleanup transactions"? I don't think we should use different terms.